crates/ra_analysis/tests/tests.rs

```diff
@@ -2,14 +2,30 @@ use ra_syntax::TextRange;
 use test_utils::{assert_eq_dbg, assert_eq_text};
 
 use ra_analysis::{
-    mock_analysis::{analysis_and_position, single_file, single_file_with_position, MockAnalysis},
+    mock_analysis::{analysis_and_position, analysis_and_range, single_file, single_file_with_position, MockAnalysis},
     AnalysisChange, CrateGraph, FileId, FnSignatureInfo,
 };
 
 fn get_signature(text: &str) -> (FnSignatureInfo, Option<usize>) {
     let (analysis, position) = single_file_with_position(text);
     analysis.resolve_callable(position).unwrap().unwrap()
 }
+#[test]
+fn test_type_of() {
+    let (analysis, range) = analysis_and_range(
```

Comment:
existing `single_file_with_range` should be enough here.

Reply:
Thanks, however, Should I remove `with_files_and_range` and `analysis_and_range` at this PR?